docs/understanding-airbyte/high-level-view.md

```diff
@@ -15,33 +15,43 @@ A more concrete diagram can be seen below:
 ```mermaid
 ---
 title: Architecture Overview
-config:
-  theme: neutral
 ---
 flowchart LR
     W[fa:fa-display WebApp/UI]
-    S[fa:fa-server Server/Config API]
+    S[fa:fa-server Config API Server]
     D[(fa:fa-table Config & Jobs)]
-    T(fa:fa-calendar Temporal)
+    L[(fa:fa-server Launcher)]
+    O[(fa:fa-superpowers Orchestrator)]
+    Q[(fa:fa-superpowers Queue)]
+    T(fa:fa-calendar Temporal/Scheduler)
     W2[1..n Airbyte Workers]
+    WL[fa:fa-server Workload API Server]
+    
     W -->|sends API requests| S
     S -->|store data| D
     S -->|create workflow| T
     T -->|launch task| W2
-    W2 -->|return job| T
-    W2 -->|launches| Source
-    W2 -->|launches| Destination
+    W2 -->|return status| T
+    W2 -->|creates job| WL
+    WL -->|queues workload| Q
+    Q  -->|reads from| L
+    L -->|launches| O
+    O -->|launches/reads from| Source
+    O -->|launches/reads from/writes to| Destination
+    O -->|reports status to| WL
```

Comment:
@Adorism any idea this looks pretty small on my browser.. any idea how to make this click & zoom?

Reply:
e.g. the rendered vercel: https://airbyte-docs-git-davinchia-worker-v2-public-airbyte-growth.vercel.app/understanding-airbyte/high-level-view

Reply:
Using a different renderer solved this!